scripts/fuzz_opt.py

```diff
@@ -423,6 +423,8 @@ def write_commands_and_test(opts):
                     commands = [in_bin('wasm-opt') + ' -all a.wasm'] + get_commands(opts)
                     write_commands(commands, 'tt.sh')
                     # reduce the input to something smaller with the same behavior on the script
+                    # wasm-reduce needs to find the binaries
+                    os.environ['BINARYEN_ROOT'] = options.binaryen_root
```

Comment:
What binaries does it need to find?    Normally the binary would know how to find the other binaries based on where it is installed.  If that is not that case and it needs other data from the source tree, why isn't that data installed as well?

Reply:
`wasm-reduce` runs at least `wasm-opt` (but I think that's it).

Oh, are you suggesting that `argv[0]` be used?

Reply:
Yes exactly.  I know the binaries should live in the same directory so that seems perfectly reasonable to me.